Chat/Application/Exceptions/ExceptionStatus.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.Exceptions
+{
+    public static class ExceptionStatus
+    {
+        public enum Status
+        {
+            NotFound = 404,
+            Unauthorized = 401,
+            BadRequest = 400
+        }
+    }
```

Comment:
Why create a static class for an enum? You can simply create the enum itself without a static class and use it